lib/tapioca/cli.rb

```diff
@@ -133,7 +138,11 @@ def dsl(*constants)
       )
 
       Tapioca.silence_warnings do
-        command.execute
+        if options[:list_compilers]
```

Comment:
I'm surprised this needs to be inside the `silence_warnings` block.

Reply:
Honestly, never even paid attention to that. It probably can be outside that block indeed.